sql.py

```diff
@@ -17,24 +17,44 @@ def user_exists(self, username):
             result = self.cursor.execute("SELECT * FROM `users` WHERE `username` = ?", (username,)).fetchall()
         return bool(len(result))
 
-    def register(self, username, name, surname, id):
+    def register(self, username, name, id, contact):
```

Comment:
fucking username

Reply:
can go without it but if a user has it(majority does), it will give an advantage for a person who is being tracked and it would be more convenient for him